docs/tables/aws_cloudfront_origin_request_policy.md

```diff
@@ -11,9 +11,8 @@ select
   name,
   id,
   comment,
-  e_tag,
-  last_modified_time,
-  type
+  etag,
```

Comment:
@Priyanka-Chatterjee-2000 why has the `type` column been removed? Is that column unavailable in the table?

Reply:
Yes `type` column does not exist in the table `aws_cloudfront_origin_request_policy`.